src/main/webapp/WEB-INF/pages/cart.jsp

```diff
@@ -0,0 +1,88 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="tags" tagdir="/WEB-INF/tags" %>
+
+<jsp:useBean id="cart" type="com.es.phoneshop.model.cart.Cart" scope="request"/>
+
+<tags:master pageTitle="Cart">
+    <c:if test="${not empty paramValues.success}">
+        <p>
+            <span style="color: green">Successfully updated</span>
+        </p>
+    </c:if>
+    <c:if test="${not empty success}">
+        <p>
+            <span style="color: red">Updating error</span>
+        </p>
+    </c:if>
+
+    <form method="post">
+        <table>
+            <thead>
+            <tr>
+                <td>
+                    Image
+                </td>
+                <td>
+                    Description
+                </td>
+                <td>
+                    Quantity
+                </td>
+                <td>
+                    Price
+                </td>
+                <td>
+                    Action
+                </td>
+            </tr>
+            </thead>
+
+            <c:forEach var="cartItem" items="${cart.cartItems}" varStatus="status">
```

Comment:
What is the cart is empty? Will we get empty table with headers?
Please consider checking for empty cart before rendering the table.